index.bs

```diff
@@ -157,57 +157,104 @@ This is often used to attempt to trick users into visiting scam websites.
-
-If a feature is powerful enough to require user consent,
-but it's impossible to explain to a typical user what they are consenting to,
-that's a signal that you may need to reconsider the design of the feature.
-
-If a permission prompt is shown,
-and the user doesn't grant permission,
-the Web page should not be able to do anything
-that the user believes they have refused consent for.
-
-By asking for consent,
-we can inform the user of what capabilities the web page does or doesn't have,
-reinforcing their confidence that <a href="#safe-to-browse">the web is safe</a>.
-However, the <a href="#priority-of-constituencies">user benefit</a>
-of a new feature must justify the additional burden on users
-to decide whether to grant permission for each feature
-whenever it's requested by a Web page.
+Using such a feature should only be possible
+if the user's expectation matches the feature's consequences
+(e.g., the personal information it reveals or the state it changes).
+
+Design your API so its behavior matches what people intend and expect.
+Then, in the best case, people don't need to get involved in allowing API access
+and thus can't make decisions they regret later. 
+Only ask the user for confirmation in exceptional circumstances.
+If a capability can be easily reverted or causes only mild annoyance without the chance for permanent harm, 
+afford users an option to stop capability access instead.
+
+<p class="example">
+For example, web pages don't need to ask for permission to play sounds.
+It is easy to stop audio output if a site is found to be abusing it,
+and browsers have a number of mechanisms for doing just that.
+</p>
+
+<h4 id="user-decisions">Help users make good decisions</h4>
+
+Empower good decision-making by limiting risk at the API level and
+providing clarity, context, and user control.
+
+Asking for user approval has been a common way to handle the remaining risk.
+However, people often can't make good decisions on
+the questions they're asked  
+because they lack crucial context and information.  
+Being confronted with difficult or annoying questions leads
+to decision fatigue, habitual responses,
+annoyance from frequent interruptions, and regrettable decisions.
+
+When user involvement is unavoidable,
+empower users to make good decisions.
+Use these principles:
+
+* **Limit the consequences from a bad decision by addressing risk at the API level first.**
+    Do not ask users questions where a "bad" decision can lead to severe consequences,
+    like a compromised device or serious personal data disclosure.
+* **Provide constant feedback about persistent decisions.**   
+    If users' decisions last longer than the current session,
+    user agents should remind users that their past decision still applies.
+    Provide a way to retract any ongoing permission.
+    APIs should include a way for sites to learn of the change in status.
+* **Ask questions people can understand.**
```

Comment:
@hober made a [request](https://github.com/w3ctag/design-principles/pull/584#discussion_r2341460179) to make this linkable. It's possible to use an `<a id=stuff>`, but probably [better to use](https://speced.github.io/bikeshed/#self-link)

```suggestion
<li id="ask-understandable-questions"> **Ask questions people can understand.**
```

Because of https://github.com/speced/bikeshed/issues/2184, you'll also have to change the other `*`s to `<li>`s and surround the list with `<ul>`. We can also do this in a separate PR after merging yours.